### ISSUE TITLE
Feature request: possibility to reverse the Num Lock state RGB indicator

### ISSUE THREAD

**Issue by Enrice** [WootingKb/wootility-issues#102]
The way the Num Lock indicator works right now is counter intuitive to me. I am used to a lit LED indicating Num Lock being active. It would be nice to have this configurable. Any fiddling with the colors (actually swapping them) is an imperfect workaround and disturbs other functionality (like some effects).

**Desktop (please complete the following information):**
 - OS: Windows 10

**Wootility and Firmware version**
wootility 4.1.2-beta, firmware 2.2.3

**Keyboard Type**
Wooting two lekker german ISO

**Comment by GsakuL** [comment]
(Wooting Two Flaretech 1.30.12 // Wootility 3.6.15 // Windows10)

I would like to add to this.

My default profile uses white/"lightgray" as the color for the whole keyboard. 
I want numlock to be enabled by default, and rarely change it.
The current impelentation however uses hardcoded white to signal NumLock=Off, which is 'invisible' on my profile.
Sometimes I hit the key, and don't notice it, until some keybinding won't work, or text is malformated.

Obviusly a two-color-option (like with FN/caps lock) would be at least better. But I don't know if there is some weird logic with animation, since currently the NumLock state is "reversed" in firmware.

Swapping the logic back and forth would only result in "one side" of the "NumLock-Fanbase" being annoyed. Thus it should be a more "in-depth" solution (in my opinion). (Like at least have a "reverse-logic-toggle", a animatabale 'main' color, and a static indicator color)

tl;dr I'd like to customize both state colors; not only reverse indicator logic

**Comment by Enrice** [comment]
That's what I basically was requesting...

> It would be nice to have this configurable

**Comment by adzenith** [comment]
> Obviusly a two-color-option (like with FN/caps lock) would be at least better.

Looks like this at least is possible now. You can pick the color of the little circle inside the key and it will change the highlight color.
e.g. here numlock is green when active:
<img width="39" alt="image" src="https://user-images.githubusercontent.com/614934/148213987-d9f9b295-6390-4d8a-85df-e82b5951342d.png">
